cel/ipv4.go

```diff
@@ -0,0 +1,182 @@
+// Copyright 2023-2024 Buf Technologies, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cel
+
+import (
+	"strconv"
+)
+
+type Ipv4 struct {
+	str       string
+	index     int64
+	strLen    int64
```

Comment:
64 bits for string length seems excessive. Can't we use whatever's typical here, probably `int`?

This applies to all parser structs.